Quintana_Jerrod/Assignments/python_fundamentals/names.py

```diff
@@ -0,0 +1,32 @@
+students = [
+     {'first_name':  'Michael', 'last_name' : 'Jordan'},
+     {'first_name' : 'John', 'last_name' : 'Rosales'},
+     {'first_name' : 'Mark', 'last_name' : 'Guillen'},
+     {'first_name' : 'KB', 'last_name' : 'Tonel'}
+]
+users = {
+ 'Students': [
+     {'first_name':  'Michael', 'last_name' : 'Jordan'},
+     {'first_name' : 'John', 'last_name' : 'Rosales'},
+     {'first_name' : 'Mark', 'last_name' : 'Guillen'},
+     {'first_name' : 'KB', 'last_name' : 'Tonel'}
+  ],
+ 'Instructors': [
+     {'first_name' : 'Michael', 'last_name' : 'Choi'},
+     {'first_name' : 'Martin', 'last_name' : 'Puryear'}
+  ]
+ }
+for index in range(len(students)):
+    string =''
+    for key,val in students[index].items():
+        string += val + ' '
+    print string
+for key,val in users.items():
+    print key
+    for i in range(len(val)):
```

Comment:
good. in cases where you don't care about the indices, it's more efficient and simpler to iterate the elements directly, e.g.: `for person in val`